Formula/gitlab-ci-multi-runner.rb

```diff
@@ -4,8 +4,8 @@ class GitlabCiMultiRunner < Formula
   desc "The official GitLab CI runner written in Go"
   homepage "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner"
   url "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner.git",
-    :tag => "v1.5.2",
-    :revision => "76fdacdf55c5a83249a4bdc5bd8c650e7a5ff796"
+    :tag => "v1.5.3",
+    :revision => "fb49c4781e5040539d3faec87a2a63675d2c9f00"
```

Comment:
For urls everything that wraps to the next line should be transitioned to align under the `"` just as you did with the go_resource blocks. The only thing that I think remains controversial is which elements should not wrap at all. I'd like to see us do one element per line regardless of which subset of elements it is. So `:shallow` gets its own line, `:branch` gets its own line, `:tag` gets its own line, `:revision` gets its own line, `:using` gets it own line. But I think @DomT4 wants to make `:branch` an except that never wraps, and I think possibly `:shallow` too, but I'm not sure. And then other exceptions creep in like if the `:revision` doesn't cause it to exceed the eightieth column then you may (must? should?) keep it on the same line, etc., as well as possibly other exceptions that have yet to be revealed.